packages/next/bin/next.ts

```diff
@@ -84,55 +78,18 @@ if (args['--help']) {
 const defaultEnv = command === 'dev' ? 'development' : 'production'
 process.env.NODE_ENV = process.env.NODE_ENV || defaultEnv
 
-const bin = join(__dirname, 'next-' + command)
-
-const startProcess = () => {
-  const proc = spawn('node', [...nodeArguments, bin, ...forwardedArgs], { stdio: 'inherit' })
-  proc.on('close', (code, signal) => {
-    if (code !== null) {
-      process.exit(code)
-    }
-    if (signal) {
-      if (signal === 'SIGKILL') {
-        process.exit(137)
-      }
-      // tslint:disable-next-line
-      console.log(`got signal ${signal}, exiting`)
-      process.exit(signal === 'SIGINT' || signal === 'SIGTERM' ? 0 : 1)
-    }
-    process.exit(0)
-  })
-  proc.on('error', (err) => {
-    // tslint:disable-next-line
-    console.error(err)
-    process.exit(1)
-  })
-  return proc
-}
-
-let proc = startProcess()
-
-function wrapper() {
-  if (proc) {
-    proc.kill()
-  }
-}
+const bin = join(process.argv[1], '../next-' + command)
 
-process.on('SIGINT', wrapper)
-process.on('SIGTERM', wrapper)
-process.on('exit', wrapper)
+process.argv = [ process.argv[0], bin, ...forwardedArgs ]
```

Comment:
I don't think we have to override `process.argv`, the `arg` package allows for passing in the argv instead of it automatically reading: https://github.com/zeit/arg#usage

Reply:
Then you couldn't run `./next-build ./path`